tools/pipelines/server-routerlicious.yml

```diff
@@ -71,6 +71,10 @@ pr:
     - server/routerlicious
     - tools/pipelines/server-routerlicious.yml
     - tools/pipelines/templates/build-docker-service.yml
+    - tools/pipelines/templates/include-generate-notice-steps.yml
+    - tools/pipelines/templates/include-publish-npm-package-deployment.yml
+    - tools/pipelines/templates/include-publish-npm-package-steps.yml
+    - tools/pipelines/templates/include-publish-npm-package.yml
```

Comment:
I don't understand our pipelines very well, and there is not much detail in the PR description, so its not clear how including these pipelines which seem related to publishing packages is related to adjusting a working directory.

Reply:
You're right, there's only one of these that's needed. I've cleaned it up and added details to the PR description.